### ISSUE TITLE
Element with role radio is missing aria-checked attribute

### ISSUE THREAD

**Issue by Reem-yahoo** [Organization-yahoo/org-repo#3]
**Rule description**  
One or more elements that has a role=�radio� attribute, excluding input type='radio' elements, not intentionally hidden in the DOM and available to assistive technologies, does not have an aria-checked attribute.

**Tool name**  
Access Engine

**Severity**  
Critical

**WCAG Success criteria**  
[4.1.2 Name, Role, Value - Level A](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)

**Why it matters**  
When the <code>aria-checked</code> attribute is not used or set to an invalid value the intended state (or any state) may not be communicated to users of assistive technology such as screen readers.

**What to do**  
Authors need to set the element's aria-checked attribute to <code>true</code>, <code>false</code>.

**Task in the Level Access Platform**  
https://flightpath.hub.dev.dev.levelaccess.io/my-tasks/697b8ec98542250bb269ea57/6ab41ebdd75f6371511df315

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access

**Comment by Reem-yahoo** [comment]
<a href="https://flightpath.hub.dev.dev.levelaccess.io/tasks/6ab41ebdd75f6371511df315/attachments/6ab41ebdd75f6371511df313/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access

**Comment by Reem-yahoo** [comment]
Task pushed to GitHub Issues by Reem Aldroubi- new dev from Level Access Platform.

Posted by Reem Aldroubi- new dev via Level Access
